rockusb/src/protocol.rs

```diff
@@ -227,6 +285,38 @@ impl CommandBlock {
         }
     }
 
+    pub fn capability() -> CommandBlock {
+        CommandBlock {
+            tag: fastrand::u32(..),
+            transfer_length: 8,
+            flags: Direction::In,
+            lun: 0,
+            cdb_length: 0x6,
+            cd_code: CommandCode::ReadCapability,
+            cd_opcode: 0,
+            cd_address: 0,
+            cd_length: 0,
+        }
+    }
+
+    pub fn erase_blocks(first: u32, count: u16, lba: bool) -> CommandBlock {
```

Comment:
Looking at how rkdeveloptool handles these; it sems this should really 2 two different functions for clarity ; Also probably better to call it `erase_force` and `erase_lba` rather then calling one `erase_blocks` to stay closer to the protocol

Reply:
Yep, that works too, makes sense to have 1 specific opcode per command